types/materialize-css/inputfields.d.ts

```diff
@@ -0,0 +1,7 @@
+/// <reference path="./common.d.ts" />
+
+declare namespace M {
+    function updateTextFields(): void;
+
+    function textareaAutoResize(textarea: Element | MElements): void;
```

Comment:
I don't think `MElements` is appropriate here. `textareaAutoResize` does not accept an `NodeListOf<Element>`

Reply:
fixed